balancer_conn_wrappers.go

```diff
@@ -151,6 +151,7 @@ func (ccb *ccBalancerWrapper) watcher() {
 			for acbw := range scs {
 				ccb.cc.removeAddrConn(acbw.getAddrConn(), errConnDrain)
 			}
+			ccb.UpdateBalancerState(connectivity.TransientFailure, nil)
```

Comment:
Should this be `Connecting` instead to avoid RPC failures?

Reply:
RPCs won't fail when picker is nil, they will block (but then the state and picker are actually inconsistent? Maybe).

I think `Connecting` is better. Done.